data/src/test/kotlin/com/antosik/benchproject/data/popular/movies/entity/remote/MovieRemoteTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.antosik.benchproject.data.popular.movies.entity.remote
+
+import com.antosik.benchproject.data.popular.movies.entity.database.MovieEntity
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.TestFactory
+
+internal class MovieRemoteTest {
+
+    @TestFactory
+    fun `SHOULD map to MovieEntity type WHEN toEntity was called on MovieRemote`() =
```

Comment:
type word is redundant here in my opinion (applicable also for the rest of the test)

Reply:
Done 😄